### ISSUE TITLE
Commands::Deploy Error: RuntimeError

### ISSUE THREAD

**Issue by kazooligan1982** [DeployGate/deploygate-cli#35]
# Status
deploygate-cli ver 0.0.4

# Error message
Not local install certificate

# Backtrace
```
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/builds/ios/set_profile.rb:105:in `install_provisioning'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/builds/ios/set_profile.rb:48:in `create_provisioning'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/commands/deploy/build.rb:141:in `create_provisioning'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/commands/deploy/build.rb:48:in `ios'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/commands/deploy/build.rb:17:in `run'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/commands/deploy.rb:16:in `run'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/command_builder.rb:37:in `block (2 levels) in run'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/runner.rb:71:in `run!'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/command_builder.rb:59:in `run'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/bin/dg:6:in `<top (required)>'
/usr/bin/dg:23:in `load'
/usr/bin/dg:23:in `<main>'
```

**Comment by henteko** [comment]
memo: not install local certificate

**Comment by henteko** [comment]
fix #43 
release v0.0.5
